addons/mail/static/src/components/message/message.scss

```diff
@@ -52,7 +52,6 @@
         // It also excludes all div in li because 1st leaf and div child of list overflow
         // may impact the bullet point (at least it does on Safari).
         max-width: 100%;
-        overflow-x: auto;
```

Comment:
Hello, this rule is necessary in other cases, in particular to prevent arbitrary email content from messing up the interface. It is not ok to just remove it.

The ideal fix would be to fix the template containing this button so that its style rules don't lead to an overflow in the first place.

Reply:
Changes Done !